src/UID2.Client/DecryptionResponse.cs

```diff
@@ -1,36 +1,30 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace UID2.Client
 {
-    public struct DecryptionResponse
+    public readonly struct DecryptionResponse
     {
-        private readonly DecryptionStatus _status;
-        private readonly string _uid;
-        private readonly DateTime? _established;
-        private readonly int? _siteId;
-        private readonly int? _siteKeySiteId;
-
-        public DecryptionResponse(DecryptionStatus status, string uid, DateTime? established, int? siteId, int? siteKeySiteId)
+        public DecryptionResponse(DecryptionStatus status, string uid, DateTime? established, int? siteId, int? siteKeySiteId, bool? isCstgDerived)
```

Comment:
This is a breaking change. Let's discuss further.

Reply:
Good point, I will create a constructor overload to avoid breaking change. Don't see why anyone would create our response but since it's public will preserve it.

Reply:
Remembered that C# has default parameter values - just defaulted this one so no need to overload